tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -6841,12 +6845,38 @@ Status ConvertGraphDefToEngine(
   TF_RETURN_IF_ERROR(statusor.status());
   std::unique_ptr<Converter> converter = std::move(statusor.ValueOrDie());
 
+  tensorflow::grappler::GrapplerItem grappler_item;
+
+  GraphDef output_graph;
+
+  grappler_item.graph = gdef;
+  const char* lo_env = std::getenv("TF_TRT_ENABLE_LAYOUT_OPTIMIZER");
+  int lo_val;
+  bool apply_layout_optim = lo_env == nullptr ||
+                          strings::safe_strto32(lo_env, &lo_val) ? lo_val == 1
+                                                                 : false;
```

Comment:
You can use ReadBoolFromEnvVar like [here](https://github.com/sourcecode369/tensorflow-1/blob/9f446aba8aaeb2b3c4c6e5ba1ab4cf31494b8a64/tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc#L2360).